dpgen/remote/RemoteJob.py

```diff
@@ -821,12 +839,17 @@ def _make_script(self,
         ret = ''
         ret += "#!/bin/bash -l\n#BSUB -e %J.err\n#BSUB -o %J.out\n"
         if res['numb_gpu'] == 0:
-            ret += '#BSUB -R span[ptile=%d]\n#BSUB -n %d\n' % (res['node_cpu'], res['numb_node'] * res['task_per_node'])
-        else :
-            ret += '#BSUB -R "select[ngpus >0] rusage[ngpus_excl_p=1]"\n#BSUB -n %d\n' % (res['numb_gpu'])
-        #ret += '#BSUB -l walltime=%s\n' % (res['time_limit'])
-        #if res['mem_limit'] > 0 :
-        #    ret += "#BSUB -l mem=%dG \n" % res['mem_limit']
+            ret += '#BSUB -R span[ptile=%d]\n#BSUB -n %d\n' % (
+                res['node_cpu'], res['numb_node'] * res['task_per_node'])
+        else:
+            ret += '#BSUB -R span[ptile=%d]\n#BSUB -n %d\n' % (
+                res['numb_node'] * res['task_per_node'], res['numb_node'] * res['task_per_node'])
```

Comment:
What the purpose of this line? It originally aims to submit a GPU task. Why not use `numb_gpu`?

Reply:
And, are you sure this script can apply for a GPU resource?

Reply:
For the test cluster I finished the test, it is of no use to add `#BSUB -R "select[ngpus >0] rusage[ngpus_excl_p=1]"` item, for the GPU hosts are configured in the same way as CPU ones. And for the original code, it would be useful if the GPU hosts were configured in the right way. So I will modify my code to support the correct situation. Thanks for your comment.